src/main/resources/THIRDPARTYREADME.txt

```diff
@@ -1,3591 +0,0 @@
-DO NOT TRANSLATE OR LOCALIZE
```

Comment:
Shouldn't we keep this file since it probably contains licenses of third party libraries? Or is it something else?

Reply:
FR removed this file between 1.0 and 1.3. I'm just mirroring exactly what's in their source JAR for 1.3.